### PR TITLE
Create theory.MD

### PR DESCRIPTION
**Pull request by yU281**

### CODE REVIEW

**Comment by yU281** [comment]
@Nelsonchris1

File: theory.MD [hunk 1]
```diff
@@ -0,0 +1,64 @@
+# foundation-homework4
+	# H1 Homework
+
+ Git is an example of a distributed version control system.
+
```
Comment (Nelsonchris1) [inline theory.MD:5]: 1/1 CORRECT

File: theory.MD [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+Q2 Github
```
Comment (Nelsonchris1) [inline theory.MD:6]: 1/1 CORRECT

File: theory.MD [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+Modified Staged Committed
```
Comment (Nelsonchris1) [inline theory.MD:8]: 3/3 CORRECT

File: theory.MD [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+ PR stands for Pull Request Repo is short for Repository
```
Comment (Nelsonchris1) [inline theory.MD:10]: 2/2 CORRECT

File: theory.MD [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+
+git pull
+
```
Comment (Nelsonchris1) [inline theory.MD:13]: 1/1 CORRECT

File: theory.MD [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+git fetch
+
```
Comment (Nelsonchris1) [inline theory.MD:15]: 1/1 CORRECT

File: theory.MD [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+git push
+
```
Comment (Nelsonchris1) [inline theory.MD:17]: 1/1 CORRECT

File: theory.MD [hunk 8]
```diff
@@ -0,0 +1,64 @@
...
+            except InsufficientFundsError:
+                print("You do not have enough money to purchase this item.")
+                additional_money = input("Do you have more money? Enter the amount: ")
+                try:
+                    additional_money = float(additional_money)
+                    customer_money += additional_money
+                except ValueError:
+                    print("Invalid amount entered. Exiting the shop.")
+                    break
+
```
Comment (Nelsonchris1) [inline theory.MD:64]: 4/ 12 : Good attempt. But this is the wrong submission format. Please if unclear about the submission format, ask one of the instructors.

**Review by Nelsonchris1 — COMMENTED** [review]
QUESTION 1: 2/2
TOTAL: 16/40
FEEDBACK: Good attempt. However, you have provided no code for testing and your submission format is incorrect. please follow the provided submission formats. Also, revisit the lecture on testing or use the provided link if you are still unclear about testing 
https://www.freecodecamp.org/news/how-to-write-unit-tests-for-python-functions/

**Pull request closed without merging**